library/vulnerabilities/ssrf/inspectDNSLookupCalls.test.ts

```diff
@@ -86,7 +88,7 @@ t.test("it blocks lookup in blocking mode", (t) => {
     "operation"
   );
 
-  runWithContext(context, () => {
+  runWithContext(getContext(), () => {
```

Comment:
is this related?

Reply:
pff okay it's fine

Reply:
Yes, otherwise the tests fail because we reuse the same context object and set forceProtectionOff.